apps/site/src/views/Home/sections/Intro/Intro.tsx

```diff
@@ -0,0 +1,34 @@
+import Image from "next/image";
+import Row from "react-bootstrap/Row";
+import pin from "@/assets/images/index-card-pin.svg";
+import hackDoodle from "@/assets/images/hack-doodle.svg";
+import styles from "./Intro.module.scss";
+
+export default function Intro() {
+	return (
+		<Row className={styles.intro + " m-5"}>
+			<Image
```

Comment:
Issue: we should still use a `Container` element to limit the maximum width on large-width views.
Consider
```tsx
<Container as="section">
	<div className={styles.card}>
		<Image
			...
```